kapitan/cli.py

```diff
@@ -657,16 +656,11 @@ def secret_update_validate(args, ref_controller):
             recipients = kap_inv_params['secrets']['gpg']['recipients']
         except KeyError:
             recipients = None
-        #
-        # Azure KMS marker -- remove it
-        #
+        # TODO: same key but in a different vault (check ..)
```

Comment:
This TODO should either be fixed before merging or be more detailed so that if needed someone else can also take over. :)